Text/HTML/TagSoup/Tree.hs

```diff
@@ -49,6 +50,11 @@ tagTree = g
             where (a,b) = f xs
         f [] = ([], [])
 
+parseTree :: StringLike str => str -> [TagTree str]
+parseTree = tagTree . parseTags
+
+parseTreeOptions :: StringLike str => ParseOptions str -> str -> [TagTree str]
+parseTreeOptions = ((.).(.)) tagTree parseTagsOptions
```

Comment:
My preference here would be not so point free. Not too fussed though, since the function is clear and simple.